examples/seismic/viscoacoustic/wavesolver.py

```diff
@@ -87,16 +98,17 @@ def forward(self, src=None, rec=None, v=None, r=None, p=None, qp=None, b=None,
         # Create all the fields v, p, r
         save_t = src.nt if save else None
         v = v or VectorTimeFunction(name="v", grid=self.model.grid, save=save_t,
-                                    time_order=1, space_order=self.space_order)
+                                    time_order=self.time_order,
+                                    space_order=self.space_order)
 
         # Create the forward wavefield if not provided
         p = p or TimeFunction(name="p", grid=self.model.grid, save=save_t,
-                              time_order=1, space_order=self.space_order,
+                              time_order=self.time_order, space_order=self.space_order,
                               staggered=NODE)
 
         # Memory variable:
```

Comment:
what is the memory variable?

Reply:
Hi George, thank you for your review! The R variable is described as memory variable on geophysics domain and is not connected with any computacional expect. This variable contain information about energy loss of stress-strain, and for containing this information is described as a memory variable.